<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Server Information</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spnServerPort">
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>12345</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnStartServer">
       <property name="text">
        <string>Start Server</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblConnectionStatus">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#lblConnectionStatus[state =&quot;0&quot;]{

	background-color: rgb(239, 41, 41);
	
}

#lblConnectionStatus[state =&quot;1&quot;]{

	background-color: rgb(138, 226, 52);
	
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="state" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>241</width>
      <height>123</height>
     </rect>
    </property>
    <property name="title">
     <string>Message Information</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="InMessage"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendToAll">
       <property name="text">
        <string>Send To All</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="ledImageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>341</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>351</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="lstConsole"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lstConsole</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
